src/timeseriesinsights/azext_timeseriesinsights/tests/latest/recordings/test_timeseriesinsights_environment_longterm.yaml

```diff
@@ -100,7 +100,7 @@ interactions:
       message: OK
 - request:
     body: 'b''{"location": "westus", "sku": {"name": "L1", "capacity": 1}, "kind":
-      "LongTerm", "properties": {"timeSeriesIdProperties": [{"name": "[\''DeviceId1\'']",
```

Comment:
I see that we're changing the way we're parsing the `timeSeriesIdProperties` here. Looks good!